src/ripple/app/paths/impl/PaySteps.cpp

```diff
@@ -423,6 +429,12 @@ toStrandV2 (
         if (hasCurrency && hasIssuer &&
             isXRP(pe.getCurrency()) != isXRP(pe.getIssuerID()))
             return {temBAD_PATH, Strand{}};
+
+        if (hasIssuer && (pe.getIssuerID() == noAccount()))
```

Comment:
No, there's no currency to check against (of course, when the strand is constructed there will be a currency to check against, and it will be checked there).